templates/analytics/remote_server_support.html

```diff
@@ -88,6 +88,26 @@ <h4>On 100% sponsored Zulip Community plan.</h4>
                 {% endif %}
             </form>
             {% endif %}
+
+            {% if plan_data[remote_server.id].current_plan %}
+            <div class="remote-server-information">
+                <h4>📅 Current plan information:</h4>
+                <b>Plan name</b>: {{ plan_data[remote_server.id].current_plan.name }}<br />
+                <b>Status</b>: {{plan_data[remote_server.id].current_plan.get_plan_status_as_text()}}<br />
+                {% if plan_data[remote_server.id].current_plan.tier != 101 %}
+                <b>Billing schedule</b>: {% if plan_data[remote_server.id].current_plan.billing_schedule == plan_data[remote_server.id].current_plan.BILLING_SCHEDULE_ANNUAL %}Annual{% else %}Monthly{% endif %}<br />
+                <b>Licenses</b>: {{ plan_data[remote_server.id].licenses_used }}/{{ plan_data[remote_server.id].licenses }} ({% if plan_data[remote_server.id].current_plan.automanage_licenses %}Automatic{% else %}Manual{% endif %})<br />
+                {% if plan_data[remote_server.id].current_plan.price_per_license %}
+                <b>Price per license</b>: ${{ plan_data[remote_server.id].current_plan.price_per_license/100 }}<br />
+                {% elif plan_data[remote_server.id].current_plan.fixed_price %}
+                <b>Fixed price</b>: ${{ plan_data[remote_server.id].current_plan.fixed_price/100 }}<br />
+                {% endif %}
+                <b>Next invoice date</b>: {{ plan_data[remote_server.id].current_plan.next_invoice_date.strftime('%d %B %Y') }}<br />
+                {% else %}
```

Comment:
This might be better intended to hint more clearly the nesting structure for the if statements.